src/cli/upload.rs

```diff
@@ -89,6 +92,7 @@ fn do_upload(buffer: Vec<u8>, asset_id: u64, cookie: &str) -> anyhow::Result<()>
         "https://data.roblox.com/Data/Upload.ashx?assetid={}",
         asset_id
     );
+	let urli = url.clone();
```

Comment:
Why are you doing this? Looks like you can just `&url`?

Reply:
if you do that you'll get a borrow after move  compile error.

i'm not a rust expert but a generally programming practice that carries over well to passing rust's helpful compiler rules is to avoid re-using variables for different purposes.   

so to fix error and consistent w/general rule i cloned it to the url info variable.  it only lives for a few microseconds so should be fine for this application imho.

Reply:
The correct solution here is not to clone the URL, it's just to move only a reference to the URL into the closure. I don't know what you mean about general programming practice, but it doesn't really apply here.